src/api/treeTrackerApi.js

```diff
@@ -523,6 +523,33 @@ export default {
       handleError(error);
     }
   },
+  getSessions() {
```

Comment:
Will want to limit results to the orgs/user's sessions only...
Which is best field to query by?
      orgList &&...
          device_configuration_id
          originating_wallet_registration_id
          target_wallet
          organization
          organization_id
          device_identifier
          grower_account_id